generator/booking_dataset.hpp

```diff
@@ -15,6 +15,9 @@ namespace generator
 class BookingDataset
 {
 public:
+  double static constexpr kDistanceLimitInMeters = 150;
+  double static constexpr kOptimalThreshold = 0.709283;
```

Comment:
Comment about how this magic constant was calculated (and how the reader may calculate it).